daffodil-lib/src/main/scala/org/apache/daffodil/lib/schema/annotation/props/Properties.scala

```diff
@@ -103,7 +103,7 @@ abstract class Enum[A] extends EnumBase with Converter[String, A] {
 
   def toPropName(prop: A) = prop.toString
 
-  val values: Array[A]
+  def values: Array[A]
```

Comment:
What's the reason for this change? Using an abstract val is nice since it forces implementations to also use a val and ensures the result is calculated only once. Does Scala 3 get rid of that?

Reply:
Hmm not sure, it was an inherited change and changing it back didn't  break anything with the compiler, so I'll leave it as a val